sim_interfaces/airsim_zenoh_bridge/mock_detection_publisher.py

```
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+﻿#!/usr/bin/env python3
```

Ruff (0.15.9)

[warning] Shebang should be at the beginning of the file

(EXE005)